packages/expo-router/src/__tests__/matchers.test.ios.ts

```diff
@@ -40,6 +41,37 @@ describe(matchGroupName, () => {
     expect(matchGroupName('leading/(foo,bar)/(fruit,apple)')).toEqual('foo,bar');
   });
 });
+describe(matchLastGroupName, () => {
+  it(`matches`, () => {
+    expect(matchLastGroupName('[[...foobar]]')).toEqual(undefined);
+    expect(matchLastGroupName('[[foobar]]')).toEqual(undefined);
+    expect(matchLastGroupName('[...foobar]')).toEqual(undefined);
+    expect(matchLastGroupName('[foobar]')).toEqual(undefined);
```

Comment:
sort undefined cases together in their own test, same with other tests. Make this more readable and predictable when things fail.